server.js

```diff
@@ -1,12 +1,10 @@
-const express = require('express')
 const http = require('http')
 const path = require('path')
 const WebSocket = require('ws')
 const chokidar = require('chokidar')
 
-module.exports = function createServer({ paths, ignored }) {
-  const app = express()
```

Comment:
Trying to remember why I even added express 😅

Reply:
There's a chance it's doing something automatic with upgrading websocket connections but I doubt it. I'll make sure I include a test making sure that websocket connections do indeed function without it

Reply:
Awesome, appreciate it! 🙏